docs/content/_navigation.json

```diff
@@ -711,6 +711,10 @@
             "title": "Pandas (dagster-pandas)",
             "path": "/_apidocs/libraries/dagster-pandas"
           },
+          {
+            "title": "Pandera (dagster-pandas)",
```

Comment:
Is this copy pasta? Should it be `dagster-pandera` here and in `path`?

Reply:
yes